src/element/types.ts

```diff
@@ -116,6 +122,8 @@ export type ExcalidrawTextElement = _ExcalidrawElementBase &
     baseline: number;
     textAlign: TextAlign;
     verticalAlign: VerticalAlign;
+    textContainerId: ExcalidrawGenericElement["id"] | null;
```

Comment:
I was thinking of renaming `textContainerId` to `containerId`. 1) we may support containers for other element types in the future 2) more importantly, the `textContainerId` may be confusing: it the container a text, or it the container containing the text (would be something others or our future selves will ask)?